website/content/en/preview/faq.md

```diff
@@ -175,6 +179,16 @@ Yes.  See [Persistent Volume Topology]({{< ref "./concepts/scheduling#persistent
 ### Can I set `--max-pods` on my nodes?
 Not yet.
 
+### Why do the Windows2019 and Windows2022 AMI families only support Windows Server Core?
+The difference between the Core and Full variants is that Core is a minimal OS with less components and no graphic user interface (GUI) or desktop experience.
+`Windows2019` and `Windows2022` AMI families use the Windows Server Core option for simplicity, but if required, you can specify a custom AMI to run Windows Server Full.
+
+You can specify the [Amazon EKS optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html) with Windows Server 2022 Full for Kubernetes 1.27 by configuring an `amiSelector` that references the AMI name.
```

Comment:
It can be a follow-up. We should consider adding a tutorial somewhere with a sample NodeTemplate for launching instances based on Full AMIs.

Launching both Core and Full should be available as a step by step so that customers do not have to do substantial work/research on their end to configure Windows specific things.

Reply:
I considered the same and I was thinking this could be addressed in an AWS blog post but you are also right that we should have easy steps someone can read and follow in Karpenter docs itself. Lets address this as a follow up.